langchain-core/src/prompts/prompt.ts

```diff
@@ -116,9 +116,19 @@ export class PromptTemplate<
 
   constructor(input: PromptTemplateInput<RunInput, PartialVariableName>) {
     super(input);
+    // If input is mustache and validateTemplate is not defined, set it to false
+    if (
+      input.templateFormat === "mustache" &&
+      input.validateTemplate === undefined
+    ) {
+      this.validateTemplate = false;
+    }
```

Comment:
cc @nfcampos thoughts on this?

Reply:
If I don't do this, you'll always need to explicitly `validateTemplate: false` in prompt options when using `mustache`

Reply:
Seems reasonable. Maybe add log a warning/info message? It's helpful to know when values are changing internally.